MerlinAU.sh

```diff
@@ -2282,12 +2365,27 @@ _CheckTimeToUpdateFirmware_()
 }
 
 ##-------------------------------------##
-## Added by Martinski W. [2024-Jan-24] ##
+## Added by Martinski W. [2024-Feb-16] ##
 ##-------------------------------------##
+_RunEMailNotificationTest_()
```

Comment:
Turned this into a callable function, as always your implementation of improvements is well received :) Thank you again!

Reply:
Tested and working.